testsuite/MDAnalysisTests/coordinates/test_dcd.py

```diff
@@ -243,17 +256,30 @@ def test_writer_dt(tmpdir, ext, decimal):
     universe_dcd = mda.Universe(PSF, DCD, dt=dt)
     t = universe_dcd.trajectory
     outfile = str(tmpdir.join("test.{}".format(ext)))
-    with mda.Writer(outfile, n_atoms=t.n_atoms, dt=dt, istart=t._file.header['istart']) as W:
+    # use istart=None explicitly so that both dcd and xtc start with time 1*dt
+    # (XTC ignores istart, DCD will set istart = nsavc)
+    with mda.Writer(outfile, n_atoms=t.n_atoms, dt=dt, istart=None) as W:
         for ts in universe_dcd.trajectory:
             W.write(universe_dcd.atoms)
 
     uw = mda.Universe(PSF, outfile)
     assert_almost_equal(uw.trajectory.totaltime,
-                        (uw.trajectory.n_frames - 1) * dt, decimal)
+                        (uw.trajectory.n_frames - 1) * dt, decimal=decimal,
+                        err_msg="Total time  mismatch for ext={}".format(ext))
     times = np.array([uw.trajectory.time for ts in uw.trajectory])
-    frames = np.array([ts.frame for ts in uw.trajectory])
-    assert_array_almost_equal(times, (frames + 1000) * dt, decimal)
+    frames = np.arange(1, uw.trajectory.n_frames + 1)  # traj starts at 1*dt
+    assert_array_almost_equal(times, frames * dt, decimal=decimal,
+                              err_msg="Times mismatch for ext={}".format(ext))
 
+@pytest.mark.parametrize("variable, default", (("istart", 0), ("nsavc", 1)))
+def test_DCDWriter_default(tmpdir, variable, default):
+    outfile = str(tmpdir.join("test.dcd"))
+    with mda.Writer(outfile, n_atoms=10) as W:
```

Comment:
In principle, writing the empty DCD and returning the header could be made a module-level fixture. I am out of time to do this right now.